docs/tutorials/pre_executed/zubercal-ps1-snad.ipynb

```diff
@@ -0,0 +1,2344 @@
+{
```

Comment:
<p>Nice to show shutting down your client at the end of the notebook, just to exhibit good hygiene for the readers.</p><p> </p><p>Can you also make some Science Statement here? like... look at the difference in detection cadence. or "that SNAD101 sure looked boring until ZTF starting observing some transient phenomena! pretty nifty!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I used <code>with Client() as client</code>, so I believe no need to shutdown it explicitly. I'm adding some words, thanks for the suggestion!</p>